packages/protocol/congestioncontrol/icca/scheduler/scheduler.go

```diff
@@ -614,6 +653,12 @@ func WithRate(rate time.Duration) options.Option[Scheduler] {
 	}
 }
 
+func WithMaxDeficit(maxDef int) options.Option[Scheduler] {
```

Comment:
This is not used anywhere. Add this parameter to `SchedulerParametersDefinition` and pass this value down.

Reply:
done